tests/test_solvers/test_harbor_solver.py

```
@@ -1037,11 +1065,13 @@ def __init__(self) -> None:
             def is_empty(self) -> bool:
                 return False
 
+        adapter_kwargs: dict[str, object] = {}
+
         class FakeAdapter:
             is_mounted = True
 
             def __init__(self, *args, **kwargs) -> None:
```

Ruff (0.15.20)

[warning] Missing type annotation for `*args`

(ANN002)